account_skip_bank_reconciliation/models/account_bank_statement.py

```diff
@@ -1,19 +1,19 @@
-# © 20118 Eficent Business and IT Consulting Services S.L. (www.eficent.com)
+# © 2018-19 Eficent Business and IT Consulting Services S.L. (www.eficent.com)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 from odoo import models
+from odoo.osv import expression
 
 
 class AccountBankStatementLine(models.Model):
     _inherit = "account.bank.statement.line"
```

Comment:
Wrong inherit here, should be:
class AccountReconciliation(models.AbstractModel):
    _inherit = 'account.reconciliation.widget'

After this the module works great!

Reply:
ops, oversight. fixed, thanks :+1: